docs/variants/novacustom_ns5x_tgl/test-matrix.md

```diff
@@ -52,6 +52,8 @@ subjected before the release of the new binary.
 | 41.  | [Sign of life][SOL]                                    | SOL           | All                                  |
 | 42.  | [ME disable/neuter support][MNE]                       | MNE           | MNE001.001                           |
 | 43.  | [USB-C docking station SD card reader][DSD]            | DSD           | All                                  |
+| 44.  | [BIOS menu function keys][BMF]                         | BMF           | All                                  |
```

Comment:
This PR should concern only on net interface

Reply:
@johanes2115 and how about that?

Reply:
Nie są to moje zmiany, przesuwałem w inne miejsce w dokumencie linijkę z pod spodu, prawdopodobniee git zczytał to jako zmiana.